### PR TITLE
added files

### PR DESCRIPTION
**Pull request by komenAnna**
### Description
Structuring a web page that is provided with content by adding structural elements to it so it can have a page layout applied to it

Fixes # (issue)

- [x] Adding structural website elements which include
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Change Details

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
The web page layout with the required website page as specified in the guidelines of the project

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gertie-sheshe — COMMENTED** [review]
🔥✨ Great job getting this done! @komenAnna  The HTML semantics are looking good!

Before we proceed, currently you have an `assets` folder and two folders inside it. 

Could you:
1. Delete the __MACOSX folder with everything in it
2. Move all the files in the second `assets` folder into the first one and the delete this folder
3. The result will be just one `assets` folder with the files inside

**Comment by gertie-sheshe** [comment]
Hey @komenAnna you can address the comments I left ✨

**Comment by komenAnna** [comment]
Will do, thanks!

**Comment by komenAnna** [comment]
I am having some trouble updating the change in my Github account. Maybe we
could discuss this in our 1:1 session?

File: assets/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            <li><a href="#">Photos</a></li>
+            <li><a href="#">Gear</a></li>
+            <li><a href="#">Forum</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <main>
+        <article>
+          <section>
```
Comment (gertie-sheshe) [inline assets/index.html:33]: You do not need this `section` element with only a `h2` in it :)

**Review by gertie-sheshe — APPROVED** [review]
Thanks for addressing the previous changes - added one more comment, when you fix that you can go ahead and merge this :) @komenAnna

**Pull request merged**